README.rst

```diff
@@ -47,16 +47,16 @@ Features
 --------
 
 - Fully-implemented, drop-in replacement for datetime
-- Supports Python 3.6+
+- Support for Python 3.6+
 - Timezone-aware and UTC by default
-- Provides super-simple creation options for many common input scenarios
-- :code:`shift` method with support for relative offsets, including weeks
-- Formats and parses strings automatically
-- Wide support for ISO 8601
+- Super-simple creation options for many common input scenarios
+- ``shift`` method with support for relative offsets, including weeks
+- Format and parse strings automatically
+- Wide support for the `ISO 8601 <https://en.wikipedia.org/wiki/ISO_8601>`_ standard
 - Timezone conversion
-- Timestamp available as a property
+- Support for ``dateutil``, ``pytz``, and ``ZoneInfo`` tzinfo objects
```

Comment:
Do the `ZoneInfo` objects work currently?

Reply:
Yup! It works out of the box since we just pass the `ZoneInfo` object to `datetime`:

```python
>>> import arrow
>>> from zoneinfo import ZoneInfo
>>> arrow.now(tz=ZoneInfo("America/Los_Angeles"))
<Arrow [2021-02-18T10:27:50.351851-08:00]>
```

The only place where dateutil enters the equation is when we are parsing tz strings or offsets. Else, we just pass the object on to `datetime`.

In my other PR, I was attempting to swap dateutil for `ZoneInfo` wherever possible internally, but we ran into that weird Alaska Nanuk issue that we wanted to ask Paul about.

Reply:
On Python <3.9, users will just need to import ZoneInfo from backports, but that is up to them (the caller) since they are the ones to form the tz object before calling into Arrow.

Reply:
Very nice!